src/adapter/graphql/src/scalars/data_query.rs

```diff
@@ -71,15 +67,6 @@ impl DataQueryResult {
         })
     }
 
-    pub fn no_schema_yet(format: DataBatchFormat, limit: u64) -> DataQueryResult {
-        DataQueryResult::Success(DataQueryResultSuccess {
-            schema: None,
-            data: DataBatch::empty(format),
-            datasets: None,
-            limit,
-        })
-    }
```

Comment:
It would be great to add a note about this in CHANGELOG

Reply:
Done